content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: New Deployment Models for Istio
+subtitle: Clear Separation between Mesh Operator and Mesh Users
+description: A new deployment model for Istio.
+publishdate: 2020-08-19
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [multicluster,install,deploy,'1.7']
+---
+
+## Background
+
+In working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+* Mesh Operator
+* Platform Owner
```

Comment:
Unclear what Platform Owner is responsible for. The rest of the blog groups platform owner and service owner persons together. Suggest to clarify what the roles of each are, or just remove one of Platform or Service

Reply:
Platform owner is the overall owner of all services and their running platform, who controls gateway resources, host/domain/ports.  service owner is owner for individual service.